README.md

```diff
@@ -1,3 +1,28 @@
+# 1. Introduction
+
+This web application is a [Google Summer of Code 2019](https://summerofcode.withgoogle.com) project under the [MacPorts](https://www.macports.org) organisation.
```

Comment:
I would probably say something more in the sense of "This web application has been implemented during [Google Summer of Code 2019](https://summerofcode.withgoogle.com) as a project mentored by [MacPorts](https://www.macports.org) organisation."

But slightly more important it might make sense to start with the "what this is" part rather than how it came to be.